src/app/util/decode_webp.cpp

```diff
@@ -42,11 +42,10 @@ os::SurfaceRef decode_webp(const uint8_t* buf, uint32_t len)
 
   WebPDecoderConfig config;
   WebPInitDecoderConfig(&config);
-  if (WebPGetFeatures(webp_data.bytes, webp_data.size, &config.input)) {
-
-  }
-  else {
-    config.input.has_alpha = false;
+  auto status = WebPGetFeatures(webp_data.bytes, webp_data.size, &config.input);
+  if (status != VP8_STATUS_OK) {
```

Comment:
@dacap since the code of this class was copy & pasted from webp_format.cpp, I think we should review the use of the result of WebPGetFeatures in that file.